go.mod

```diff
@@ -31,6 +32,7 @@ require (
 	github.com/mdlayher/vsock v1.1.1
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.19.0
+	github.com/opencontainers/image-spec v1.0.2-0.20190823105129-775207bd45b6
```

Comment:
Why such an old version? podman uses `go.mod: github.com/opencontainers/image-spec v1.0.3-0.20220114050600-8b9d41f48198`, `go get -u github.com/opencontainers/image-spec` switches to the `v1.0.2` tag which was created in 2021.

Reply:
@cfergeau I am not sure https://github.com/opencontainers/image-spec here I don't even find v1.0.3 tag or branch.

Reply:
https://github.com/opencontainers/image-spec/issues/918 there is request for release.

Reply:
> I don't even find v1.0.3 tag or branch

```
$ go get -u github.com/opencontainers/image-spec@main
go: downloading github.com/opencontainers/image-spec v1.0.3-0.20220512140940-7b36cea86235
go: upgraded github.com/opencontainers/image-spec v1.0.2 => v1.0.3-0.20220512140940-7b36cea86235
```
It's go which refers to a non existent 1.0.3 version when we ask it to use the `main` branch. But as I said in my earlier comment, using the latest release (`1.0.2`) is also fine with me.

Looking again at this, it seems the 20190823105129 is misleading, and that `v1.0.2-0.20190823105129-775207bd45b6` might be newer than v1.0.2, but history is complicated with a lot of merges, so not sure :-/